Documentation/dev-guide/interacting_v3.md

```diff
@@ -355,6 +355,23 @@ foo         # key
 bar_latest  # value of foo key after modification
 ```
 
+## Watch progress
+
+Applications may want to check the progress of a watch to determine how up-to-date the watch stream is. For example, if a watch is used to update a cache, it can be useful to know if the cache is stale compared to the revision from a qoroum read. 
```

Comment:
`s/qoroum/quorum/` :)

Reply:
Can we also clarify this revision in watch response to progress notify request *is* revision from the local node that the watch stream is connected to? Since quorum read cannot be returned from the partitioned node, mention something like "stale compared to the revision fetched from a healthy cluster (e.g. header in quorum read)".

Reply:
Ah, yes, I should be more explicit about that revision is returned. Fixing.